test/martha/martha_v3.test.js

```diff
@@ -745,34 +761,28 @@ test.serial('martha_v3 parses a Gen3 CRDC CIB URI response correctly', async (t)
 
 // BT-236 temporarily cut access token and access endpoint out of the flow
 test.serial('martha_v3 parses BDC response correctly', async (t) => {
+    const drsHost = config.HOST_BIODATA_CATALYST_STAGING;
+    const {
+        id: objectId, self_uri: drsUri,
+        access_methods: { 0: { access_id: accessId, access_url: { url: gsUrl } } }
+    } = bdcDrsResponse;
```

Comment:
NOTE: This destructuring of metadata responses will not work for `self_uri` from `hcaDrsResponse` and some others. (Yay "specifications"!) But those tests can be modified to hard code the `drsUri` some other way.